src/react/components/Gateway.tsx

```diff
@@ -0,0 +1,37 @@
+import { useAnchorWallet, useConnection } from "@solana/wallet-adapter-react";
+import React from "react";
+import { Badge, GatewayStatus, useGateway } from "@civic/solana-gateway-react";
+import { PublicKey } from "@solana/web3.js";
+import { config } from "../../config";
+import { CivicButton } from "@components/buttons/CivicButton";
+
+interface Props {
+	gatekeeperNetwork: PublicKey;
+	className?: string;
+}
+
+export const Gateway = (props: Props) => {
```

Comment:
I would rename this to something more appropriate. This component isn't a Gateway. Just two civic related components